src/js/components/Chart/stories/Zoom.js

```diff
@@ -14,7 +13,7 @@ const intervalDays = {
   '1 year': 365,
 };
 
-const ZoomChart = ({ data, max }) => {
+export const ZoomChart = ({ data, max }) => {
```

Comment:
Is this export truly needed?
```suggestion
const ZoomChart = ({ data, max }) => {
```

Reply:
I guess not,
Thanks for updating that! ♥